typescript/sdk/src/timelock/evm/constants.ts

```diff
@@ -1,5 +1,8 @@
 import { keccak256 } from 'ethers/lib/utils.js';
 
+export const EMPTY_BYTES_32 =
```

Comment:
Surprised we don't have a const for this already?

Reply:
technically we do have it but it is called `COSMOS_NATIVE_ZERO_ADDR` or `STARKNET_ZERO_ADDR` so i'd say the name doesn't actually match the current use case ahahaa